### PR TITLE
Fix ML feebacks from QA

### PR DESCRIPTION
**Pull request by mfrachet**
This should fix:

- Asset uploading / Editing
- Cropping preview
- Carousel that changes its slides when using the keyboard in a dialog appearing over the carousel

### CODE REVIEW

File: packages/core/upload/server/controllers/validation/upload.js [hunk 1]
```diff
@@ -19,7 +19,7 @@ const multiUploadSchema = yup.object({
 const validateUploadBody = (data = {}, isMulti = false) => {
   const schema = isMulti ? multiUploadSchema : uploadSchema;
 
-  return validateYupSchema(schema)(data);
+  return validateYupSchema(schema, { strict: false })(data);
```
Comment (mfrachet) [inline packages/core/upload/server/controllers/validation/upload.js:22]: ℹ️ Fixes the backend for upload, thanks to @petersg83

File: packages/core/upload/admin/src/components/MediaLibraryInput/Carousel/CarouselAssets.js [hunk 2]
```diff
@@ -24,65 +25,89 @@ export const CarouselAssets = ({
...
+              )}
+            >
+              <CarouselAsset asset={asset} />
+            </CarouselSlide>
+          ))
+        )}
+      </Carousel>
+
+      {isEditingAsset && (
+        <EditAssetDialog
```
Comment (mfrachet) [inline packages/core/upload/admin/src/components/MediaLibraryInput/Carousel/CarouselAssets.js:91]: ℹ️ Shifting the dialog outside the carousel fixes the keyboard bug when being on an input

File: packages/core/upload/admin/src/components/EditAssetDialog/PreviewBox/index.js [hunk 3]
```diff
@@ -73,11 +73,11 @@ export const PreviewBox = ({
     const nextAsset = { ...asset, width, height };
     const file = await produceFile(nextAsset.name, nextAsset.mime, nextAsset.updatedAt);
 
-    await editAsset(nextAsset, file);
+    const updatedAsset = await editAsset(nextAsset, file);
 
     // Making sure that when persisting the new asset, the URL changes with width and height
     // So that the browser makes a request and handle the image caching correctly at the good size
-    const optimizedCachingImage = createAssetUrl(asset);
+    const optimizedCachingImage = createAssetUrl(updatedAsset);
```
Comment (mfrachet) [inline packages/core/upload/admin/src/components/EditAssetDialog/PreviewBox/index.js:80]: ℹ️ createAssetUrl uses the "updated_at" attribute to handle image caching. The problem is that I was using the previous asset and so, the URL was always the same => cache was hit in this scenario

**Review by soupette — APPROVED** [review]
LGTM

**Review by HichamELBSI — APPROVED** [review]
LGTM good job @mfrachet !

**Pull request merged**